pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-8270-composite-add-separator.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
```

Comment:
We'll likely need to update the other Liquibase migrations at some point, but `https` should be used here now.

```suggestion
                   xsi:schemaLocation="https://www.liquibase.org/xml/ns/dbchangelog
                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
```